package/com.unity.formats.usd/Documentation~/USD Import.md

```diff
@@ -0,0 +1,53 @@
+# Importing USD Files
+
+![](Images/usd-import-options.PNG)
+
+To import a USD file into Unity, use the USD context menu, and select one of the 'Import as ...' options as required. Supported USD file formats are:
+
+-   .USD
+-   .USDA
+-   .USDC
+-   .USDZ
+
+## Importing Options
+
+-   **Import As GameObjects**: Imports selected USD file as GameObjects directly into the Scene Hierarchy.
```

Comment:
Would be good to elaborate on the benefits of the GameObjects vs Prefab option? As a 'new user' I'm really not clear which option should be my go-to option to start with. Will I find a limitation from one vs the other etc.